docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -196,6 +196,7 @@ jprob = JumpProblem(repressilator, dprob, Direct(), save_positions=(false,false)
 # now, let's solve and plot the jump process:
 sol = solve(jprob, SSAStepper(), saveat=10.)
 plot(sol)
+plot(sol, plotdensity = 1000, fmt = :png) # hide
```

Comment:
Can we update the preceding lines to not use `save_positions` or `saveat` since with this they aren’t needed?

Reply:
good point, yes removing that